### DISCUSSION TITLE
Eva Logik ZW31 seems to be supported in the devices config, but comes up in ZWave To MQTT 6.6.3 as unknown?

### DISCUSSION THREAD

**Discussion by uberDoward** [zwave-js/zwave-js#2130] [Q&A] [answered]
_**tl;dr:**_

Unknown manufacturer 0x0312 
and
Unknown product 0xaa02 

But they seem to be in the device configs for node-zwave-js?

_**Ok, longer:**_
It was purchased as a Eva Logik ZW31 S2 3 way dimmer off Ye Olde Amazon.  Started looking into how to update the device configurations (I'm a .NET dev by day, new to the home automation game, though) and came across [the overview for the config files](https://zwave-js.github.io/node-zwave-js/#/config-files/overview) - yay!

I see the productId and the hex all seems to align in [zw31.json](https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/devices/0x0312/zw31.json) so wondering if I need to force some kind of database update from the configs, or maybe they aren't packaged yet?  I did a compare in Github of 6.6.2 --> 6.6.3, and 6.6.3 --> 7.0.0-beta2, but the files _seem_ like they should have been already included in the release?

So what might I be missing?  TIA!

**Comment by AlCalzone** [comment, marked as answer]
Please open an issue (bug report) for this and fill out the template. I'll need to see a logfile where you re-interview the node in question (please wait with triggering the re-interview until all has settled).

The problem might be subtle, so all I can do here is guess.

**Reply by uberDoward** [reply]
Will do!  Got a production deploy tonight, so will write it up then.
